klarna-on-demand/build.gradle

```diff
@@ -1,4 +1,6 @@
 apply plugin: 'com.android.library'
+apply plugin: 'maven'
+apply plugin: 'signing'
 
 android {
```

Comment:
I have actually copied extensively from that plugin, yes. What I have here is actually the result of mashing together that plugin, what braintree do, Sonatype's guide and some random blogpost.

Stripe went ahead and used that directly, but if you go look [here](https://github.com/stripe/stripe-android/blob/master/stripe/build.gradle#L30) I think you'll get why I'm not so keen on using the plugin directly.